_data/task.yml

```
@@ -9,8 +9,10 @@ websites:
     img: airtable.png
     tfa:
       - sms
+      - phone
       - totp
     doc: https://support.airtable.com/hc/en-us/articles/219409917
+    exception: "Software token can be bypassed by SMS."
```

I think @Jawshy and I both captured something the other missed. "SMS-capable phone required" is insufficient because it only (unambiguously) communicates that you need a SMS-capable phone to set it up. It doesn't reflect the interaction between software & SMS.

```suggestion
    exception: "SMS must be enabled first and software 2FA can be bypassed by SMS."
```

---

@ilovecats516, would you like to commit djfredalert's suggestion?